docs/core/install/linux-package-mixup.md

```diff
@@ -50,104 +54,155 @@ Mixing .NET packages from two different sources will most likely lead to issues
 
         ```bash
         Package: dotnet* aspnet* netstandard*
         Pin: origin "packages.microsoft.com"
         Pin-Priority: -10
         ```
 
-    01. Install .NET.
+    01. Reinstall .NET from the distribution's package feed. For more information, see [Install .NET on Linux](linux.md).
+
+  - **No**
+
+    01. Remove the existing .NET packages from your distribution. You want to start over and ensure that you don't install them from the wrong repository.
 
         ```bash
-        sudo apt-get update && sudo apt-get install -y dotnet7
+        sudo apt remove 'dotnet*' 'aspnet*' 'netstandard*'
         ```
 
-03. **I need a recent version of .NET that's not provided by the Linux distribution repositories.**
+    01. Delete the Microsoft repository feed from your distribution.
+
+        ```bash
+        sudo rm -f /etc/apt/sources.list.d/microsoft-prod.list
+        sudo apt update
+        ```
+
+    01. Reinstall .NET from the distribution's package feed. For more information, see [Install .NET on Linux](linux.md).
+
+::: zone-end
+
+### I need a version of .NET that isn't provided by my Linux distribution
 
-    In this case, keep the Microsoft repository, but configure it so .NET packages from the Microsoft repository are considered a higher priority. Then, remove the already-installed .NET packages and then re-install the .NET packages from the Microsoft repository.
-  
-    For Fedora, CentOS 8+, RHEL 8+, use the following bash commands:
+::: zone pivot="os-linux-redhat"
+
+Configure your package manager to ignore the .NET packages from the distribution's repository. It's possible that you've installed .NET from both repositories, so you want to choose one or the other.
+
+01. Remove the existing .NET packages from your distribution. You want to start over and ensure that you don't install them from the wrong repository.
 
     ```bash
-    echo 'priority=50' | sudo tee -a /etc/yum.repos.d/microsoft-prod.repo
     sudo dnf remove 'dotnet*' 'aspnet*' 'netstandard*'
-    sudo dnf install dotnet-sdk-7.0
     ```
 
-    For Ubuntu (or any other distribution that uses apt) make sure you've installed the Microsoft repository feeds and then install the package.
+01. Configure the Microsoft repository to ignore .NET packages.
 
-    01. Remove the .NET packages if you previously installed them. For more information, see scenario #1.
-    01. Create `/etc/apt/preferences` if it doesn't already exist.
-    01. Add the following config to the preferences file, adding the specific package you want to pin as a high priority, which will be sourced by the Microsoft Feed. For example, use the following example to pin .NET SDK 7.0:
+    ```bash
+    echo 'excludepkgs=dotnet*,aspnet*,netstandard*' | sudo tee -a /etc/yum.repos.d/ <TODO>WHAT IS THE REPO NAME HERE??</TODO> 
```

Comment:
Is this still a TODO?

Reply:
AHHHH Thanks!!

@omajid Do you have an idea on how to do this??

Reply:
Changing this to a generic statement.

Reply:
So, this a bit fuzzy, but here's what I see when I try and download .NET from Microsoft

- The "Install .NET on Linux instructions" for  .NET 7 link on `dot.net` takes me to https://learn.microsoft.com/en-ca/dotnet/core/install/linux?WT.mc_id=dotnet-35129-website
- That says ".NET is available in [official package archives](https://github.com/dotnet/core/blob/main/linux.md) for various Linux distributions and [packages.microsoft.com](https://packages.microsoft.com/)"
- https://packages.microsoft.com/ points to https://packages.microsoft.com/docs/readme.txt
- The readme says "When possible, use the configuration files under /config and use standard Linux package managers."
- https://packages.microsoft.com/config/ points to https://packages.microsoft.com/config/fedora/36/ which then points to https://packages.microsoft.com/config/fedora/36/packages-microsoft-prod.rpm (and https://packages.microsoft.com/config/fedora/36/prod.repo)
- That file packages-microsoft-prod.rpm installs /etc/yum.repos.d/microsoft-prod.repo

So, at the end, if the users are following recommendations fully, they end up with a `/etc/yum.repos.d/microsoft-prod.repo` on disk.

But it's possible they downloaded https://packages.microsoft.com/config/fedora/36/prod.repo and installed it with a custom name, which is impossible to guess.